mayastor/tests/common/ms_exec.rs

```diff
@@ -172,14 +172,19 @@ impl MayastorProcess {
     }
 
     /// terminate the mayastor process and wait for it to die
-    pub fn sig_term(&self) {
+    pub fn sig_term(&mut self) {
+        if self.child == 0 {
```

Comment:
These changes weren't mentioned in the description - it looks like it avoids attempting to kill the same process twice?

Reply:
Yes, when it shuts down normally it calls sig_term again.

Reply:
Ah, right.